### DISCUSSION TITLE
Issue with session after signInWithOAuth

### DISCUSSION THREAD

**Discussion by hyochan** [supabase/supabase#15461] [Questions]
I've gone through https://supabase.com/docs/guides/auth/auth-helpers/nextjs#authentication several times and I am not sure where the problem is but after migrating to `@supabase/auth-helpers-nextjs": "^0.7.2"` from `@supabase/auth-helpers-nextjs": "0.6.1"`, I am getting null session after `signInWithOAuth`.

Here is my latest update https://github.com/hyochan/github-stats/commit/86fb0bd8105d64b461cb3d14f3578f9ab3d6460f#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519. It never signs in user with github provider.

When I rollback to https://github.com/hyochan/github-stats/commit/d1a00f3b7417a4c7b0c9fdc05ad4e7bb2f413756, the problem is gone.

I am not sure what I am missing 🤔

**Comment by hyochan** [comment]
I was confused. This is still a problem.

**Comment by j4w8n** [comment]
did you reference the migration guide?

https://supabase.com/docs/guides/auth/auth-helpers/nextjs#migrating-to-v07x
